armi/nuclearDataIO/rzflux.py

```diff
@@ -146,7 +146,7 @@ def _rw2DRecord(self):
         Zones are blocked into multiple records so we have to block or unblock
         them.
 
-        _rwMatrix reverses the indices into FORTRAN data order so be
+        rwMatrix reverses the indices into FORTRAN data order so be
```

Comment:
Worth considering https://numpy.org/doc/stable/reference/generated/numpy.frombuffer.html
That would read into a 1-D, then https://numpy.org/doc/stable/reference/generated/numpy.reshape.html#numpy.reshape to reshape. It even takes an arg for column-major or row major.

Reply:
great idea. Speed does kinda matter when we're reading big scatter matrices and stuff.